code/client/src/sdk/entities/human/c_human_script.cpp

```diff
@@ -44,16 +44,15 @@ namespace SDK {
     }
 
     bool C_HumanScript::GetInvulnerabilityByScript() const {
-        return m_pHuman->m_bDemigod;
+        return m_pHuman->m_bInvulnerable;
```

Comment:
@Greavesy1899 @Segfaultd can you confirm?

Reply:
I have a doubt. I have to check